tests/pipeline/test_factor.py

```diff
@@ -1283,3 +1287,24 @@ class F(Factor):
             f.to_workspace_value(pipeline_output, pd.Index([0, 1])),
             column_data,
         )
+
+
+class TestSpecialCases(WithEquityPricingPipelineEngine,
+                       ZiplineTestCase):
+
+    def check_equivalent_terms(self, terms):
```

Comment:
should you just check that DailyReturns() is Return(window_length=2)?

Reply:
The `is` won't be true, b/c `DailyReturns` is a subclass.

Reply:
This would be an interesting use-case for graph rewrite rules if we ever add them.